drivers/SmartThings/matter-evse/src/embedded_cluster_utils.lua

```diff
@@ -0,0 +1,55 @@
+local clusters = require "st.matter.clusters"
+local utils = require "st.utils"
+
+--These clusters are not available in lua_libs-v10
+clusters.ElectricalPowerMeasurement = require "ElectricalPowerMeasurement"
```

Comment:
I think these clusters should be wrapped in an if statement so that they are only included for lua libs <= 10. In matter switch for example:
https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/77064ab01f6e5eb8268048a85267cfee744ad241/drivers/SmartThings/matter-switch/src/embedded-cluster-utils.lua#L6

Reply:
Okay I'll fix it